cmake/kokkoskernels_tribits.cmake

```diff
@@ -0,0 +1,186 @@
+INCLUDE(CMakeParseArguments)
+INCLUDE(CTest)
+
+IF (KOKKOSKERNELS_HAS_TRILINOS)
+INCLUDE(TribitsETISupport)
+ENDIF()
+
+MESSAGE(STATUS "The project name is: ${PROJECT_NAME}")
+
+MACRO(KOKKOSKERNELS_PACKAGE_POSTPROCESS)
+  IF (KOKKOSKERNELS_HAS_TRILINOS)
+    TRIBITS_PACKAGE_POSTPROCESS()
+  ELSEIF(NOT KOKKOSKERNELS_HAS_PARENT)
+    INCLUDE(CMakePackageConfigHelpers)
+    configure_package_config_file(cmake/KokkosKernelsConfig.cmake.in
+                                  "${KokkosKernels_BINARY_DIR}/KokkosKernelsConfig.cmake"
+                                  INSTALL_DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/cmake/KokkosKernels)
```

Comment:
On some systems (e.g. kokkos-dev-2) when I install kokkos, the appended path to `${CMAKE_INSTALL_PREFIX}` is `lib64/cmake/KokkosKernels` instead of `lib/cmake/KokkosKernels`. Is a change needed here to make this consistent between the two packages?

Reply:
Yeah, we should probably use the same trick with GNUInstallDirs